### DISCUSSION TITLE
lingbos-sussy-mod(Fabric)

### DISCUSSION THREAD

**Discussion by Minemetero** [lingbopro/lingbos-sussy-mod#4] [General]
Here is for fabric version of lingbos-sussy-mod

**Comment by Minemetero** [comment]
![image_2024-09-03_171837637](https://github.com/user-attachments/assets/89a9314e-7f04-4718-a877-8dcc4a83d82b)
已经成功复现两个物品

**Reply by lingbopro** [reply]
nice:)

**Reply by lingbopro** [reply]
说起来我有一个早就做好的附魔一直没push :(

主要是有穿墙特性（

**Reply by Minemetero** [reply]
![image_2024-09-03_190641840](https://github.com/user-attachments/assets/fba99902-119e-416e-8854-d8a8d6094736)
又复现了一个

**Reply by Minemetero** [reply]
你猜猜房子咋没的一个角: )

**Reply by lingbopro** [reply]
> 你猜猜房子咋没的一个角: )

不喜欢炸的对吧（

**Reply by Minemetero** [reply]
![image_2024-09-03_204448926](https://github.com/user-attachments/assets/238a7e60-4909-42d7-b706-0e31b467d9fa)
已经全部复现

**Reply by Minemetero** [reply]
接下来还有些杂碎的事情要写

**Reply by lingbopro** [reply]
:)

**Reply by lingbopro** [reply]
说起来Modrinth上fabric是和forge共用project还是单独开

**Reply by Minemetero** [reply]
其实你的一些机制我还没看到

**Reply by Minemetero** [reply]
可能你有的我没有

**Reply by Minemetero** [reply]
Wiki可能还要单独写qwq

**Comment by Minemetero** [comment]
beta-2.0上线复原程度

**Reply by Minemetero** [reply]
你觉得有必要单独给你的东西分一类吗，就是创造物品栏

**Reply by Minemetero** [reply]
我觉得没啥必要 我就不做了

**Reply by Minemetero** [reply]
你那个lingbo_block我现在在用石头

**Reply by Minemetero** [reply]
石头咋样 你那个Lingbo_Block 就咋样

**Reply by Minemetero** [reply]
你看看吧 然后我就要加工作流 README 打包 然后发正式版了

**Reply by lingbopro** [reply]
> 你那个lingbo_block我现在在用石头

什么石头

**Reply by Minemetero** [reply]
```java
new Block(FabricBlockSettings.copyOf(Blocks.STONE)));
```

**Reply by lingbopro** [reply]
> ```java
> new Block(FabricBlockSettings.copyOf(Blocks.STONE)));
> ```

哦
彳亍
回头我给这个方块单独写一个物品类

**Reply by lingbopro** [reply]
话说没readme

**Reply by Minemetero** [reply]
你看看还要搞些啥

**Reply by Minemetero** [reply]
你觉得有必要单独给你的东西分一类吗 就是你做了个独立的放你的物品

**Reply by lingbopro** [reply]
> 你觉得有必要单独给你的东西分一类吗 就是你做了个独立的放你的物品

创造模式标签页吗

我觉得可以，毕竟以后应该会做挺多东西的

**Reply by Minemetero** [reply]
Coin放食品 Lingbo_Block 放建筑方块 Like和Dislike放功能性物品

**Reply by Minemetero** [reply]
我觉得也没啥不好的

**Reply by lingbopro** [reply]
> Coin放食品 Lingbo_Block 放建筑方块 Like和Dislike放功能性物品

我forge版就是，放自己标签页的同时也放原版标签页

**Reply by Minemetero** [reply]
啊？

**Reply by lingbopro** [reply]
> 啊？

（其实好像只有硬币是同时存在于我的标签页和原版食物标签页

**Reply by Minemetero** [reply]
我复原了

**Reply by Minemetero** [reply]
我可以自豪地宣布100%完成了Fabric化

**Reply by Minemetero** [reply]
我是全部都做到了放自己标签页和放原版标签页

**Reply by lingbopro** [reply]
ooo nice

**Reply by lingbopro** [reply]
那Modrinth呢

**Reply by Minemetero** [reply]
什么Modtinth

**Reply by Minemetero** [reply]
你来决定

**Reply by Minemetero** [reply]
我可以现在就开个账户

**Reply by Minemetero** [reply]
你改改你的README把Fabric给加进去

**Reply by lingbopro** [reply]
ok，我觉得可以放一个project，两个也行

**Reply by lingbopro** [reply]
我先把我做的新附魔push上来

**Reply by lingbopro** [reply]
![IDEA，启动！](https://github.com/user-attachments/assets/f673aa6f-f843-4e2e-9802-4f601196fbf4)

IDEA，启动！

**Reply by Minemetero** [reply]
2023.2给我看无语了

**Reply by Minemetero** [reply]
等一下 你没在用Community Version?

**Reply by lingbopro** [reply]
> 等一下 你没在用Community Version?

em

**Reply by lingbopro** [reply]
> 2023.2给我看无语了

之前搞的，不想升了

**Reply by Minemetero** [reply]
> > 等一下 你没在用Community Version?
> 
> em

你买了?

**Reply by lingbopro** [reply]
![image](https://github.com/user-attachments/assets/5dafdf61-a7cb-49be-9681-8d7fc6376bd9)

每次配置项目都这么慢。。。

**Reply by lingbopro** [reply]
> > > 等一下 你没在用Community Version?
> > 
> > 
> > em
> 
> 你买了?

没，但是大概23年的时候搞到了一个码

**Reply by Minemetero** [reply]
什么！

**Reply by Minemetero** [reply]
怎么搞到的:(

**Reply by Minemetero** [reply]
不过我的Community version够香了

**Reply by lingbopro** [reply]
> 怎么搞到的:(

忘了，不过那个码现在已经用不了了，我现在用的激活脚本

**Reply by Minemetero** [reply]
这个和Community Version有什么区别吗

**Reply by lingbopro** [reply]
> 这个和Community Version有什么区别吗

有应该还是有，不过对我来说好像也没啥大区别（主要是我不常写Java

**Reply by lingbopro** [reply]
滚回社区版太麻烦了，就没滚

**Reply by Minemetero** [reply]
我知道的唯一区别是可以对Github Action进行支持

**Reply by lingbopro** [reply]
> 我知道的唯一区别是可以对Github Action进行支持

那倒确实

**Reply by lingbopro** [reply]
> > 我知道的唯一区别是可以对Github Action进行支持
> 
> 那倒确实

不过GitHub Actions的话VSCode更香:)

**Reply by lingbopro** [reply]
新附魔push了，不过还不算新版本

**Reply by Minemetero** [reply]
啊？

**Reply by Minemetero** [reply]
这个PNG是不是出问题了

**Comment by Minemetero** [comment]
新版本push完了

**Reply by Minemetero** [reply]
结合进了附魔

**Reply by lingbopro** [reply]
oh

**Reply by Minemetero** [reply]
你下一步搞啥

**Reply by Minemetero** [reply]
什么时候发1.1

**Reply by Minemetero** [reply]
> oh

就一个oh那么冷漠啊

**Reply by lingbopro** [reply]
> > oh
> 
> 就一个oh那么冷漠啊

因为想不到说啥了，我现在正在搞别的东西:D

**Reply by Minemetero** [reply]
搞啥呢

**Reply by Minemetero** [reply]
我要去搞凌波蛋了

**Reply by Minemetero** [reply]
你看咋样
